core/src/main/java/org/apache/carbondata/core/datamap/DataMapFilter.java

```diff
@@ -72,6 +85,17 @@ public Expression getNewCopyOfExpression() {
     }
   }
 
+  public DataMapFilter(SegmentProperties properties, CarbonTable table, Expression expression) {
+    this.expression = expression;
```

Comment:
call above constructor this(expression, table, false)

Reply:
done